.ado/templates/react-native-init.yml

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
Steps from `yarn install` to `MSBuild` seem repetitive.
Why not use `templates/build-rnw` within this template instead?

Reply:
The steps are somewhat unique - it needs to install the CLI and RNW from a file path, then build the CLI app.

---
In reply to: [328924431](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r328924431) [](ancestors = 328924431)